docs/src/pages/components/navbar.js

```diff
@@ -14,6 +14,7 @@ import {
   ListItem,
   Text
 } from "@nulogy/components";
+import { theme } from "@nulogy/components/src";
```

Comment:
`theme` is exported by `@nulogy/components`, so you should be able to add it to the list above.